src/main/webapp/script.js

```diff
@@ -37,57 +37,57 @@ function cleanseUrl(url) {
   // TODO: Add checks to make this work if video is not first parameter.
   var videoId = url.split("?");
   videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
-
+ 
   // If param name present, remove it to isolate video Id.
   videoId = videoId.replace("v=", "");
   
   return videoId;
 }
-
+ 
 /**
  * Fetches data and adds to html
  */
 async function getChart() {
   $('form').submit(async function() {
     commentStats = await getYouTubeComments();
     averageScore = commentStats.averageScore;
-    aggregateValues = commentStats.aggregateValues; 
-
+    averageMagnitude = commentStats.averageMagnitude;
+    sentimentBucketList = commentStats.sentimentBucketList; 
+ 
     const CommentSentimentTable = new google.visualization.DataTable();
-    CommentSentimentTable.addColumn('number', 'InclusiveStart');
-    CommentSentimentTable.addColumn('string', 'SentimentRange');
-    CommentSentimentTable.addColumn('number', 'CommentCount');
-
-    // The json keys (ranges of scores) are sorted through their starting values
-    Object.keys(aggregateValues).forEach(function(key) {
-      var inclusiveStart = getRangeInclusiveStart(key);  
-      var exclusiveEnd = getRangeExclusiveEnd(key);
-      CommentSentimentTable.addRow([inclusiveStart, inclusiveStart + ' to ' + exclusiveEnd, aggregateValues[key]]);  
-    });
+    CommentSentimentTable.addColumn('string', 'Sentiment Range');
+    CommentSentimentTable.addColumn('number', 'Comment Count');
+    CommentSentimentTable.addColumn({type: 'string', role:'tooltip', 'p': {'html': true}});
+ 
+    for(i = 0; i < sentimentBucketList.length; i++) {
+      currentSentimentBucket = sentimentBucketList[i];
+      rangeAsString = currentSentimentBucket.intervalRange.asString;
```

Comment:
Forgot to push Range.java - I was planning on having a string variable (asString) in the range class that contains the object in string format, so I can call Range.asString. Would it be better to do convertRangeToString function in javascript instead? Either is fine with me, just wondering if/why one is better than the other.

Reply:
Ah, this makes a lot more sense then.

I think it feels more natural to have the `Range` -> human readable String conversion live entirely on the front end since the backend shouldn't necessarily be concerned with the UI represented to users.

As some intuition for why this is useful, imagine if we wanted this website to be usable by users who speak other languages. It would likely feel burdensome to have to pass language information to the backend so that the `asString` property could be updated w/ the correct language. It would likely be easier to just have the front end do the proper representation since it will be the closest to information about which language the user users.